portfolio/frontend/README.md

```diff
@@ -1,4 +1,35 @@
-# Getting Started with Create React App
+# SGONKS - Trading Game
+
+SGONKS is a trading game created with Node.js with Server in Java.
+
+## Table of Contents
+
+- [About the Project](#general-info)
```

Comment:
did you forget the corresponding general-info  tag?

Reply:
fix here 9acb7ad04cc0d22ebd09636ced7b845c55048b93